src/components/TranslateBoxes.js

```diff
@@ -47,19 +82,7 @@ export default function TranslateBoxes({input, handleInputChange, output, errors
                 </Grid>
                 <Grid item xs={6}>
                     <Box className = {classes.box} bgcolor="grey.200">
-                        <TextField
-                        id="standard-multiline-static"
-                        label="Translation"
-                        multiline
-                        fullWidth
-                        // TODO: create more contrast in output text font color
-                        InputProps={{
-                            style: {fontFamily: "monospace"},
-                            disableUnderline: true
-                        }}
-                        value={errors && output && addErrorsToOutput(output, errors)}
-                        disabled
-                        />
+                        {errors && output && createOutputTypography(output, errors)}
```

Comment:
I'd rather leave this `errors` and `output` checking/fallback to the function itself.